app/code/core/Mage/ConfigurableSwatches/Helper/Data.php

```diff
@@ -135,6 +135,5 @@ public function getSwatchesProductJs()
                 }
             }
         }
-        return '';
```

Comment:
A correct way is to `return null;`. But this works too.

Also this change is correct, returning `null` (by avoiding the `return` statement) will avoid to deal with an empty javascript filename. Which is better.

The impact is in `\Mage_Page_Block_Html_Head::getCssJsHtml`;

:+1:

Reply:
Really better of course ! I've updated my PR. Thanks !